### PR TITLE
Added game search endpoint to the IGDB.

### PR DESCRIPTION
**Pull request by eslong**

### CODE REVIEW

File: src/server/routes.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const request = require('request')
 
 // API keys
 const OMDBKEY = '5914db23'
+const IGDBKEY = 'a9d78f2b6558758665178eb9676390d2'
```
Comment (eatorres) [inline src/server/routes.js]: What does IGDB stand for?

Comment (eslong) [reply src/server/routes.js]: Internet Gaming Database. It's the API we are using (at least for now) for games.

Comment (eatorres) [reply src/server/routes.js]: Can you leave a comment saying what they are or put more descriptive names?

Comment (eslong) [reply src/server/routes.js]: Done, pushed in latest commit.

**Pull request merged**